### PR TITLE
Optimize write and MPP framework

### PR DESCRIPTION
**Pull request by xingtanzjr**
## Description
1. Remove some duplicated log
2. Fix the issue that `getStatus()` will be queued for a long while (50-70ms)
3. Fix the issue that the thread is occupied when sleeping in QueryTerminator, which lead to the task in same thread-pool will be queued for a long while to waiting the thread being released.
4. The write operation is changed to Sync using client thread. (The testing is in progress and the async write won't be discard)

After this change, the write throughput in 1C1D is similar with new-standalone.

### CODE REVIEW

File: server/src/assembly/resources/conf/iotdb-engine.properties [hunk 1]
```diff
@@ -1020,6 +1020,14 @@ timestamp_precision=ms
 # Datatype: int
 # datanode_schema_cache_size=10000
 
+# thread pool size for read operation in DataNode's coordinator.
+# Datatype: int
+# coordinator_read_executor_size=30
+
+# thread pool size for write operation in DataNode's coordinator.
+# Datatype: int
+# coordinator_read_executor_size=30
```
Comment (HTHou) [inline server/src/assembly/resources/conf/iotdb-engine.properties:1029]:
```suggestion
# coordinator_write_executor_size=30
```

File: server/src/assembly/resources/conf/iotdb-engine.properties [hunk 2]
```diff
@@ -1020,6 +1020,14 @@ timestamp_precision=ms
 # Datatype: int
 # datanode_schema_cache_size=10000
```
Comment (HTHou) [inline server/src/assembly/resources/conf/iotdb-engine.properties:1021]:
```suggestion
# datanode_schema_cache_size=1000000
```

File: server/src/assembly/resources/conf/iotdb-engine.properties (hunk 1 above)
Comment (xingtanzjr) [reply server/src/assembly/resources/conf/iotdb-engine.properties]: Fixed

File: server/src/assembly/resources/conf/iotdb-engine.properties (hunk 2 above)
Comment (xingtanzjr) [reply server/src/assembly/resources/conf/iotdb-engine.properties]: Fixed

**Review by HTHou — APPROVED** [review]
LGTM

**Pull request merged**